manage

```diff
@@ -476,14 +476,9 @@ $app"
     #Set terminal title
     echo -ne "\e]0;${action^}ing ${app}\a" | sed 's/Updateing/Updating/g'
     
-    #refresh the list in queue-viewer window as a background process - skip it if the list is still refreshing from last loop iteration; in game dev this is 'dropped input'
-    if [ -z "$write_list_pid" ];then
-      write_list "$queue" &
-      write_list_pid=$!
-    elif ! process_exists "$write_list_pid" ;then
-      write_list "$queue" &
-      write_list_pid=$!
-    fi
+    #refresh the list in queue-viewer window as a background process
+    write_list "$queue" &
+    write_list_pid=$!
```

Comment:
The risk with this change is that when many apps are refreshed concurrently, it spawns a new write_list background process for every app - a thread bomb. My system slowed to a standstill when refreshing 30+ apps, and the yad list was blank and unresponsive until close to the end of the 100-app stress test.

Reply:
ideally we could kill any PID that is not the most recent one waiting.

I just ran my own simulation and it got through tons very quickly but then yad coredumped 😆

Reply:
for now I have dropped that commit untill a better solution can be made